src/main/resources/swagger/agora.yaml

```diff
@@ -162,6 +162,48 @@ paths:
             - openid
             - email
             - profile
+  /api/v1/methods/{namespace}/{name}/configurations:
+    get:
+      summary: List configurations associated with a method
+      description: |
+        Given the namespace/name of a method, returns all configurations
+        in the repository that reference that method
+      parameters:
+        - name: namespace
+          in: path
+          description: Namespace of method.
+          required: true
+          type: string
+        - name: name
+          in: path
+          description: Name of method.
+          required: true
+          type: string
+      tags:
+        - Methods
+        - Configurations
+      responses:
+        '200':
+          description: An array of configurations.
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/MethodResponse'
```

Comment:
I think the should be an array of ConfigurationResponse.
I also noticed in the definition of ConfigurationResponse that it says the payload will be in wdl format. are we now returning this as json for configs?

Reply:
That seems like copypasta from the definition for `MethodResponse`.

Method payloads are WDL, config payloads have the I/O mapping & method ref.

This is the pull request to support JSON for configs: broadinstitute/agora/pull/206